src/harness/testcases/WINNF_FT_S_PAT_testcase.py

```diff
@@ -0,0 +1,124 @@
+#    Copyright 2017 SAS Project Authors. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+from datetime import datetime
+import json
+import logging
+import os
+from os import listdir
+from os.path import isfile, join
+import sas
+import sas_testcase
+
+from util import winnforum_testcase, configurable_testcase, writeConfig, loadConfig, QueryPropagationAntennaModel
+  
+def cbsddata(device):
+    installationParam = device['installationParam']
+    if 'antennaAzimuth' not in installationParam:
+        installationParam['antennaAzimuth'] = None
```

Comment:
Why?

Reply:
If not present, then it is an omni antenna. Same with antennaBeamwidth.
I recommend that you add a check for the antennaBeamwidth as well.

And possibly for the antennaGain as well: If not present, it shoud be 0